src/app/file/file.cpp

```diff
@@ -905,7 +905,7 @@ void FileOp::postLoad()
       std::shared_ptr<Palette> palette(
         render::create_palette_from_sprite(
           sprite, frame_t(0), sprite->lastFrame(), true,
-          nullptr, nullptr, m_config.newBlend));
+          nullptr, nullptr, m_config.newBlend, MapAlgorithm::RGBA));
```

Comment:
I think we need a `MapAlgorithm::DEFAULT` because here we are using for third time (or maybe more) a default value.